### ISSUE TITLE
Image seed pasted when sending to img2img or inpainting tab

### ISSUE THREAD

**Issue by Gushousekai195** [lllyasviel/stable-diffusion-webui-forge#2359]
Whenever I send a generated image to img2img or inpainting using the buttons underneath the result image, the exact seed of the generation gets sent, too. This did not happen in previous iterations of the UI and is not desirable.

Image generation parameters also change when sending the image, especially when sending them back into the img2img or inpainting tabs.

**Comment by wcole3** [comment]
See discussion in linked pull requests.  After it's merged the `send_seed` option in config.json should control the Seed paste behavior.

If you feel there are other generation parameters that should have configurable behavior describe them here.

**Comment by Gushousekai195** [comment]
How do I git pull a pull request again?

**Comment by wcole3** [comment]
```
git fetch origin pull/2360/head:respect-send-seed
git switch respect-send-seed
```

**Comment by ChooChoo38** [comment]
In addition to sending the seed, it's also turning off scripts and appending LoRA's to the prompt in a case where you wouldn't want it too. Examples:

1) I have the Ultimate SD Upscale script turned on in the img2img tab. Clicking "Send image and generation parameters to img2img tab" sets Scripts to none, disabling it. I have to re-enable it every time.

2) I add a LoRA via the "Add network to prompt" dropdown. This appends it to the prompt when you generate so you don't have to manually add it to the prompt text box yourself. However, when I send the image to the img2img tab, it appends to LoRA text to the prompt. It shouldn't do that in this case, given you've already got it enabled in the dropdown.

**Comment by ChooChoo38** [comment]
Unsure if this is related, so I'll post it here instead of the request.

In addition to sending the seed, it's also turning off scripts and appending LoRA's to the prompt in a case where you might not want it to. Examples:

1. I have the Ultimate SD Upscale script turned on in the img2img tab. Going to txt2img and clicking "Send image and generation parameters to img2img tab" sets "Script" to none in the img2img tab, disabling it. I have to re-enable it every time.

2. I add a LoRA via the "Add network to prompt" dropdown. This appends it to the prompt when you generate so you don't have to manually add it to the prompt text box yourself. However, when I send the image to the img2img tab, it automatically adds the LoRA text to the prompt itself. It shouldn't do that in this case, given I've already got it enabled in the dropdown.

**Comment by wcole3** [comment]
Thanks, almost certainly related. For 2) when pasting the Lora gets added to the prompt AND is still selected in the drop-down? Or is it just added to the prompt. 

I will take a look in a couple hours.

**Comment by wcole3** [comment]
Alright this behavior should be reverted. Apologies for the impact.

**Comment by wcole3** [comment]
Yes, that was mentioned in the PR. Behavior should be reverted now.
